### DISCUSSION TITLE
Access to data on filtered sets of paths

### DISCUSSION THREAD

**Discussion by wwahammy** [plausible/analytics#1468] [Anything Else]
First off, Plausible is AMAZING. I just stumbled on it this week and we're almost assuredly switching from Matomo to Plausible.io, it's not only more cost effective for us but it just seems easier to work with and easier to be compliant with privacy laws.

I was interested the idea of creating embeds that focus on a specific set of pages. We run a (mostly) open source fundraising platform for nonprofits so we have nonprofits creating pages on our domain. Occasionally, we get requests from the fundraising folks at the nonprofits asking if they could get access to their visitor stats. We generally tell them we can tell them but there's no way to give them access. I'm wondering if we can do that in Plausible. In particular, I'd like to let the nonprofits see data on just their pages and no other pages on our site.

I'm sure this is possible using the API but that is, of course, a significant integration challenge on our end. What I would prefer though is a way to embed this information in the nonprofit's admin dashboard on our site; it'd certainly add more value to our service. I don't think the embed has support for that use-case but perhaps this is something that would be useful for others?

**Comment by metmarkosaric** [comment]
thanks @wwahammy! 

we have the [embed dashboard](https://plausible.io/docs/embed-dashboard) feature so basically as long as you keep the pages of the individual nonprofits on their own individual Plausible dashboards (say by tracking a subdomain or perhaps a subfolder), only those pages will show up in the embed and nothing else. it's not possible to isolate and embed only a few pages from one website unfortunately. 

our [stats API](https://plausible.io/docs/stats-api) allows you more granularity and flexibility on how you can take your Plausible stats and build custom dashboards within your UI.
